src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
+  const row = table.rows[i];
+  const cells = row.cells[1].innerHTML;
```

Comment:
Code Quality Issue: The variable name `cells` is misleading, as it suggests that the variable contains multiple cells, while it actually contains the inner HTML of a single cell. A more appropriate name could be `cellContent` or `cellInnerHTML`.